### PR TITLE
adding apple cheesecake recipe

### PR DESCRIPTION
**Pull request by lakshmirangarajan**
adding `02-02-2017 applecheesecake.md`

### CODE REVIEW

File: _posts/02-02-2017-Applecheesecake.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+1/4 cup sugar
+1 tablespoon Gold Medal™ all-purpose flour
+1 teaspoon vanilla
+1 egg
+1 1/4 cups apple pie filling with more fruit, chopped (from 21-oz can)
+1/2 teaspoon ground cinnamon
+
+## Directions
+1. Heat oven to 350°F. Grease 9-inch glass pie plate with shortening.
+2 In large bowl, stir together cookie mix and 1 1/2 teaspoons cinnamon. Add 1/2 cup softened butter; mix with pastry blender or fork until crumbly. Press half (about 2 cups) of the crumbly mixture in bottom and up side of pie plate. Bake 10 minutes.
```
Comment (brianamarie) [inline _posts/02-02-2017-Applecheesecake.md]: These numbers aren't looking quite right. Maybe try adding a `.` after the digit?

File: _posts/02-02-2017-Applecheesecake.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title:  "Apple Cheesecake"
+date:   2017-02-02
+categories: main
+---
+
+![food](http://images-gmi-pmc.edge-generalmills.com/357f1106-e2a4-4df6-8543-2abdc043948a.jpg)
```
Comment (brianamarie) [inline _posts/02-02-2017-Applecheesecake.md:8]: It's OK not to include an image, but this image isn't rendering quite right.

**Review by brianamarie — CHANGES_REQUESTED** [review]
Hi @lakshmirangarajan! There are a few things not quite right with the numbers. BUT, this recipe looks DELICIOUS!!! 😋

File: _posts/02-02-2017-Applecheesecake.md (hunk 2 above)
Comment (lakshmirangarajan) [reply _posts/02-02-2017-Applecheesecake.md:8]: @brianamarie  I jus tried and it did render :+1:

Comment (lakshmirangarajan) [inline _posts/02-02-2017-Applecheesecake.md]: @brianamarie  I jus tried and it did render :+1:

**Comment by brianamarie** [comment]
@lakshmirangarajan I'm going to go ahead and make these changes and merge so you can follow along. :)

**Pull request merged**